docs/template-vm.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: linux-vm-template
+  annotations:
+    description: "OCP kubevirt linux, template"
+    tags: "kubevirt,ocp,template,linux"
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine
+  metadata:
+    name: linux-vm-${NAME}
```

Comment:
I actually wonder how the name should be.

As a user I'd input "greenhorn" as the NAME - Then, the vm linux-vm-greenhorn will be created. Makes sense. but do we want this behavior, or that a vm called "greenhorn" is getting created?

Reply:
good question, I'm open to all alternatives :)

Reply:
on one hand, it will make the name,pvcs and maybe other attributes in the future more consistent (a consistent suffix)
on the other hand we do limit the user here with a predefined prefix (unless he edits the yaml completely) 
The former is what we have in the user guide for now

Reply:
I think that the name of the `OfflineVirtualMachine` created from the template should be exactly what the user provides in the `NAME` variable:

```yaml
name: ${NAME}
```

Otherwise we will have to explain to users that the names of virtual machines are somehow calculated from the name of the template and the value that they provider for `NAME`. It can be quite confusing for users.

Reply:
no problem, changing accordingly (both templates)